src/Main.java

```diff
@@ -1,7 +1,53 @@
 import java.time.LocalDate;
 
 public class Main {
-    public static void main(String[] args) {
+    public static void calculateYear(int year) {
+        if (year > 1584) {
+            if ((year % 4 == 0 && year % 100 != 0) || (year % 400 == 0)) {
```

Comment:
```suggestion
            if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
```
лишние скобки. Также весь метод можно упростить до одного if и else, не разделяя на до 1584 или после - в любом случае, если до, то год невисокосный